.github/workflows/ci-notebook.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     - name: Install dependencies
       run: |
         pip install -U pip wheel
-        pip install --requirement requirements/devel.txt --pre --upgrade --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+        pip install -e .[dev] --pre --upgrade --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
```

Comment:
I am not fun of this in ci as this means also install the package itself, right?

Reply:
Yeah, we could just install the requirements.
Do you know why the docs tests are failing?

Reply:
> I am not fun of this in ci as this means also install the package itself, right?

Yes. Not a strong opinion at all, but why not?

`pip install -r requirements/devel.txt` only looks at `requirements/devel.txt` while `pip install -e .[dev]` looks at both `setup.py` and `requirements/devel.txt` files, right?